rpi-eeprom-update

```diff
@@ -34,6 +34,10 @@ VCMAILBOX=${VCMAILBOX:-/opt/vc/bin/vcmailbox}
 CM4_ENABLE_RPI_EEPROM_UPDATE=${CM4_ENABLE_RPI_EEPROM_UPDATE:-0}
 RPI_EEPROM_UPDATE_CONFIG_TOOL="${RPI_EEPROM_UPDATE_CONFIG_TOOL:-raspi-config}"
 
+# Automatic, critical updates are not applied unless the current bootloader version
+# is older than pieeprom-2020-09-03
+BOOTLOADER_AUTO_UPDATE_MIN_VER="${BOOTLOADER_AUTO_UPDATE_MIN_VER:-1599135103}"
```

Comment:
From a "separation of concerns" PoV, I can't decide whether or not it'd be better to have this in e.g. a config file (or symlink or something) in the `$FIRMWARE_IMAGE_DIR` directory, rather than having this `1599135103` value hardcoded in the `rpi-eeprom-update` script. Or if that'd be too fragile / error-prone.
 :confused:

Reply:
It can already be overridden via /etc/default/rpi-eeprom-update or by setting the environment variable first. I don't think it merits a separate mechanism.

Reply:
Ahh, I'd forgotten that `/etc/default/rpi-eeprom-update` gets sourced directly by the script, so can be used to override any of the variables' default values :+1:

Reply:
Yep :)